docs/source/about.rst

```diff
@@ -25,7 +25,7 @@ Citation
 
 To cite PROJ in publications use:
```

Comment:
This section would actually be better to just include from `CITATION`:

https://github.com/OSGeo/proj.4/blob/0a78cc67fc06d2fe1769c5582ce5b8b92b1ef4a3/CITATION#L1-L14

Maintaining this in two places is error-prone.

Reply:
Done, and I simplified one step of HOW-TO-RELEASE